app/views/admin/reports/registered_adviser/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2>Registered Advisers</h2>
+
+<p>
+  This report shows all advisers whose FCA status is no still 'Active'.
+</p>
+
+<p>
+  The <a href="https://register.fca.org.uk/">Financial Services Register</a>
+  can be searched to determine exactly what status change has occurred.
+</p>
+
+<% if @registered_advisers.present? %>
+  <p>
+    <%= link_to 'Download as CSV', admin_reports_registered_adviser_index_path(format: :csv), download: 'registered_advisers.csv', class: 'btn btn-primary' %>
```

Comment:
I'd like to see us move away from long lines, ideally to 80 chars, but this is open to agreement by the team.